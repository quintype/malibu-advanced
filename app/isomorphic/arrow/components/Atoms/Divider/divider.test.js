import React from "react";
import { shallow } from "enzyme";
import { Divider } from ".";

describe("Divider", () => {
  it("should render", () => {
    const wrapper = shallow(<Divider />);
    expect(wrapper).toMatchSnapshot();
  });
});
