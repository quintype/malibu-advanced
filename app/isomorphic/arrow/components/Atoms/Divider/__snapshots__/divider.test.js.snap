// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should render 1`] = `
<div
  className="divider-m_light__w-7sy"
>
  <svg
    height="10"
    width="1"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 0h1v10H0z"
      fill="inherit"
    />
  </svg>
</div>
`;
